.github/workflows/lint.yml

```diff
@@ -3,12 +3,17 @@ name: Lint
 on:
   pull_request:
     paths:
-      - src/**
-      - .clang-format
+    - .clang-format
+    - .prettierrc.json
+    - src/**
   push:
     branches:
     - main
-
+    paths:
```

Comment:
I think we want to format all on any push to main, regardless of paths

Reply:
You're right, thanks.